pkgr.yml

```diff
@@ -0,0 +1,28 @@
+Version: 1
+Descriptions:
+  - DESCRIPTION
+
+Packages:
+  - devtools
+
+Repos:
+  - MPN: https://mpn.metworx.com/snapshots/stable/2020-03-24
+  - MPN_Rcpp: https://mpn.metworx.com/snapshots/stable/2020-04-18
+  - MPN_withr: https://mpn.metworx.com/snapshots/stable/2020-05-09
+
+Customizations:
+  Repos:
+    - MPN:
+        Type: source
+    - MPN_Rcpp:
+        Type: source
+  Packages:
+    - Rcpp:
+        Repo: MPN_Rcpp
+        Type: source
+    - withr:
+        Repo: MPN_withr
+        Type: source
+
+Lockfile:
```

Comment:
I don't really understand how this works. And I assume this is what is telling pkgr where to install? I might need a brief tutorial just so I understand this code better, but it all seems to work the way it should. Well done.

Reply:
This `Lockfile` statement tells pkgr to use an renv-style (project-specific) library.